pkgs/tools/security/zmap/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libjson, json_c, gengetopt, flex, byacc, gmp
+, libpcap
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zmap";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "zmap";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0yaahaiawkjk020hvsb8pndbrk8k10wxkfba1irp12a4sj6rywcs";
+  };
+
+  patches = [ ./0001-The-byacc-binary-in-nixpkgs.byacc-is-named-yacc.patch ];
+
+  cmakeFlags = [ "-DRESPECT_INSTALL_PREFIX_CONFIG=ON" ];
+  dontUseCmakeBuildDir = true;
+
+  nativeBuildInputs = [ cmake pkgconfig gengetopt flex byacc ];
+  buildInputs = [ libjson json_c gmp libpcap ];
+
+  outputs = [ "out" "man" ];
+
+  meta = with stdenv.lib; {
```

Comment:
`platforms` is not set. It appears to support a wider variety.

Reply:
Unfortunately I don't have the resources and experience to fix broken darwin builds. I actually hoped that darwin would've worked out of the box and wanted to wait for ofborg's results.

My suggestion is that we restrict the build to linux platforms for now and let the issue fix someone with more experience here, if the demanded :)

Reply:
Oh no, I don't expect you to do anything outside your means.

I think you can still set platforms but mark it as broken for darwin.